web/js/projection/projection.js

```diff
@@ -0,0 +1,16 @@
+export function parse(state, errors, config) {
```

Comment:
It looks like wv.proj.js was exporting `proj: { parse: func()... }`, and I think this is just exporting `parse: func()...`. Will that cause problems with existing references?